plugins/builds/triggers/helpers.js

```diff
@@ -686,8 +686,8 @@ function getSubsequentJobs(workflowGraph, startNode) {
     const nodeToEdgeDestsMap = Object.fromEntries(nodes.map(node => [node.name, []]));
 
     let start = trimJobName(startNode);
-    // In rare cases, WorkflowGraph and startNode may have different start tildes
 
+    // In rare cases, WorkflowGraph and startNode may have different start tildes
```

Comment:
Thanks for finding the error.
As other members have said, it looks like a bug to have job names in `name`, `src`, `dest` with `~` prefixes other than `~commit` and `~pr`.
(I have actually encountered such a case during my last PR, as this comment.)

Also, I think it is a bug that there is a`src` or `dest` with a name that is not in the `nodes` like `certify-db`, apart from the presence or absence of `~` prefix.

Reply:
>         ### entry below comes from two remote pipelines
>         ### is this possible?
>         {
>             src: '~sd@123:deploy-api',
>             dest: 'sd@456:mysql'
>         }
Regarding the above comment in the PR context, I think that it is possible if 123 or 456 is the same as the current pipeline ID.
Conversely, if neither is the same as the current pipeline ID, it would be strange for them to exist in the this workflowGraph, because they are not related to the current pipeline at all.